data/json/items/armor/suits_protection.json

```diff
@@ -844,6 +844,18 @@
     "looks_like": "touring_suit",
     "color": "dark_gray",
     "covers": [ "LEGS", "TORSO", "ARMS" ],
+    "coverage_data": [
+      {
+        "parts": [ "arm_l", "arm_r" ],
+        "coverage": 25,
+        "encumbrance": 5
+      },
+      {
+        "parts": [ "torso", "leg_l", "leg_r" ],
+        "coverage": 95,
+        "encumbrance": [ 12, 25 ]
+      }
+    ],
     "coverage": 95,
     "encumbrance": 12,
     "max_encumbrance": 25,
```

Comment:
Why do you still have this data? It should be defined in the data above.

Reply:
This was test data, I had forgotten to remove it, thanks for pointing it out. (It should ignore those values)